fdbcli/TenantCommands.actor.cpp

```diff
@@ -985,6 +971,9 @@ void tenantGenerator(const char* text,
 	} else if (tokens.size() == 3 && tokencmp(tokens[1], "getId")) {
 		const char* opts[] = { "JSON", nullptr };
 		arrayGenerator(text, line, opts, lc);
+	} else if (tokens.size() == 4 && tokencmp(tokens[1], "list")) {
```

Comment:
We should print these even after the 4th token, since you could specify multiple of them